metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
kibana-index.json still exists in the docker stuff. Given that the module is buried in the Ambari stuff, do we still need/want the instructions and an elasticdump done to update that?